pkgs/servers/pulseaudio/default.nix

```
@@ -31,23 +35,27 @@
 
 stdenv.mkDerivation rec {
   pname = "${if libOnly then "lib" else ""}pulseaudio";
-  version = "14.2";
+  version = "15.0";
 
   src = fetchurl {
     url = "http://freedesktop.org/software/pulseaudio/releases/pulseaudio-${version}.tar.xz";
-    sha256 = "sha256-ddP3dCwa5EkEmkyIkA5FS4s1DsqoxUTzSIolYqn/ZvE=";
+    sha256 = "sha256-pAuIejupjMJpdusRvbZhOYjxRbGQJNG2VVxqA8nLoaA=";
```

Not sure whether that is a matter of personal taste, but AFAIK keeping the `sha256` as a prefix to the hash string is frowned upon.

```suggestion
    sha256 = "pAuIejupjMJpdusRvbZhOYjxRbGQJNG2VVxqA8nLoaA";
```